dotcom-rendering/src/palette.ts

```diff
@@ -45,20 +45,140 @@ const pillarPalette = (
 	}
 };
 
-const headlineColourLight: PaletteFunction = ({ design }) => {
-	switch (design) {
-		case ArticleDesign.Feature:
-			return sourcePalette.news[300];
+const headlineColourLight: PaletteFunction = ({ design, display, theme }) => {
```

Comment:
I think we should name this `headlineText` rather than `headlineColour`. All of the palette strings are colours already, so that naming is redundant.